pkg/pointer/pointer.go

```diff
@@ -19,6 +19,13 @@
 
 package pointer
 
+import v1 "kubevirt.io/api/core/v1"
+
 func P[T any](t T) *T {
 	return &t
 }
+
+func ErrorPolicyPtr(p v1.DiskErrorPolicy) *v1.DiskErrorPolicy {
+	policy := p
+	return &policy
+}
```

Comment:
I am probably a bit new to generics in golang, but why do we need an explicit function for `v1.DiskErrorPolicy`? Is it the case that `func P[T any](t T) *T` does not work?